lib/chef/win32/security.rb

```diff
@@ -668,7 +668,7 @@ def self.logon_user(username, domain, password, logon_type, logon_provider)
         Token.new(Handle.new(token.read_pointer))
       end
 
-      def test_and_raise_lsa_nt_status(result)
+      def self.test_and_raise_lsa_nt_status(result)
```

Comment:
What's the reason for changing this to a class method?

Reply:
All the other methods are class methods, we never make instances of this class, so this has to be a class method to be called.